app/src/forms/minesattestations/routes.js

```diff
@@ -14,13 +14,45 @@ routes.post('/', async (req, res, next) => {
   await controller.create(req, res, next);
 });
 
-routes.post('/submission', async (req, res, next) => {
+routes.get('/submissions', async (req, res, next) => {
+  await controller.allSubmissions(req, res, next);
+});
+
+routes.post('/submissions', async (req, res, next) => {
   await controller.createSubmission(req, res, next);
 });
 
-routes.get('/submission/:confirmationId', async (req, res, next) => {
+routes.put('/submissions/:confirmationId', async (req, res, next) => {
```

Comment:
Don't know if we care, but "Confirmation ID" was generally referring to the shortform of the guid that the user gets. So this is really the "ID" primary key right? Not the "Confirmation ID".
Minor semantic so not concerned but think about whether we want to call the REST identifier for the submission "confirmation Id"? Its the submissionId in the table right?

Reply:
yeah, confirmationId is just the short submissionId, but I use that in the API for grabbing a submission.  shorter...  although i guess no one ever sees it... 🤷‍♂️